src/NuGet.Core/NuGet.Packaging/Signing/Timestamp/Rfc3161TimestampTokenInfoNetstandard21Wrapper.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Security.Cryptography.Pkcs;
+using System.Security.Cryptography;
+
+namespace NuGet.Packaging.Signing
+{
+#if IS_SIGNING_SUPPORTED && NETSTANDARD2_1
+    public class Rfc3161TimestampTokenInfoNetstandard21Wrapper : IRfc3161TimestampTokenInfo
+    {
+        private System.Security.Cryptography.Pkcs.Rfc3161TimestampTokenInfo _rfc3161TimestampTokenInfo = null;
+
+        public Rfc3161TimestampTokenInfoNetstandard21Wrapper(byte[] timestampTokenInfo)
+        {
+            bool success = System.Security.Cryptography.Pkcs.Rfc3161TimestampTokenInfo.TryDecode(
+                new ReadOnlyMemory<byte>(timestampTokenInfo),
+                out _rfc3161TimestampTokenInfo,
+                out int bytesConsumed);
+        }
+
+        public Rfc3161TimestampTokenInfoNetstandard21Wrapper(System.Security.Cryptography.Pkcs.Rfc3161TimestampTokenInfo timestampTokenInfo)
```

Comment:
Thanks! But since we have another class with the same name Rfc3161TimestampTokenInfo, under namespace NuGet.Packaging.Signing.
I suppose adding prefix is the only way if we want to address the one under namespace System.Security.Cryptography.Pkcs.